### PR TITLE
Android API-29 enumification and stabilization

### PR DESCRIPTION
**Pull request by gugavaro**
API-29 is now Final.

### CODE REVIEW

File: build-tools/enumification-helpers/enum-conversion-mappings.xml [hunk 1]
```diff
@@ -1138,4 +1138,68 @@
...
+  <map package='android.graphics.fonts' class='FontStyle' prefix='FONT_SLANT_' enum-name="FontSlant" api-level="29" />
+  <map package='android.graphics' class='HardwareRenderer' prefix='SYNC_' enum-name="Sync" api-level="29" />
+  <map package='android.graphics' class='Paint' prefix='CURSOR_' enum-name="Cursor" api-level="29" />
+  <map package='android.graphics.text' class='LineBreaker' prefix='BREAK_STRATEGY_' enum-name="BreakStrategy" api-level="29" />
+  <map package='android.graphics.text' class='LineBreaker' prefix='HYPHENATION_FREQUENCY_' enum-name="HyphenationFrequency" api-level="29" />
+  <map package='android.graphics.text' class='LineBreaker' prefix='JUSTIFICATION_MODE_' enum-name="JustificationMode" api-level="29" />
+  <map package='android.hardware.biometrics' class='BiometricManager' prefix='BIOMETRIC_' enum-name="BiometricCode" api-level="29" />
+  <map package='android.hardware.camera2.params' class='RecommendedStreamConfigurationMap' prefix='USECASE_' enum-name="UsecaseMode" api-level="29" />
+  <map package='android.icu.lang' class='UScript' fields="*" enum-name="UScriptCode" api-level="24" />
```
Comment (jpobst) [inline build-tools/enumification-helpers/enum-conversion-mappings.xml]: A few of these are not api-level 29.  Were these just enums that we have missed in the past?  Does adding them break our existing public API?  I think we will just mark the constant fields as `[Obsolete]`, but if we updated any methods using them that is a potential breakage.

Comment (gugavaro) [reply build-tools/enumification-helpers/enum-conversion-mappings.xml]: Thank @jpobst for your review.

All remaining-int-* lines were checked but they don't have any enum associated with it. Would definitely be nice to have a way to check it as verified and removed from the file. We should discuss some options to achieve it.

Regards the non api-level 29 items, they were showing as new lines (green on the diff) so I checked, found an enum and add it. You are bringing a good point related to break the api. What is the normal workflow on those cases? don't add the enum?

let me know your thoughts.

**Comment by grendello** [comment]
There's a build error on mac:

```
[2019-06-25T20:09:44.315Z]   /Users/builder/jenkins/workspace/xamarin-android-pr-pipeline-release/xamarin-android/src/Mono.Android/Test/Mono.Android-Tests.csproj : error XA0001: Unsupported or invalid $(TargetFrameworkVersion) value of 'v9.0'. Please update your Project Options.
[2019-06-25T20:09:44.315Z] 
[2019-06-25T20:09:44.315Z] 
[2019-06-25T20:09:44.315Z] "/Users/builder/jenkins/workspace/xamarin-android-pr-pipeline-release/xamarin-android/Xamarin.Android-Tests.sln" (default target) (1) ->
[2019-06-25T20:09:44.315Z] "/Users/builder/jenkins/workspace/xamarin-android-pr-pipeline-release/xamarin-android/src/Mono.Android/Test/Mono.Android-Test.Library/Mono.Android-Test.Library.csproj" (default target) (39) ->
[2019-06-25T20:09:44.315Z]   /Users/builder/jenkins/workspace/xamarin-android-pr-pipeline-release/xamarin-android/src/Mono.Android/Test/Mono.Android-Test.Library/Mono.Android-Test.Library.csproj : error XA0001: Unsupported or invalid $(TargetFrameworkVersion) value of 'v9.0'. Please update your Project Options.
```

**Comment by jonpryor** [comment]
I'd look into the "API Breakage Question," if I could *see* the API breakage, which I can't, because the PR didn't build (?!): https://jenkins.mono-project.com/job/xamarin-android-pr-pipeline-release/1464/console

```
xamarin-android/build-tools/create-pkg/create-pkg.targets(33,5): error MSB3030: Could not copy the file "/Users/builder/jenkins/workspace/xamarin-android-pr-pipeline-release/xamarin-android/bin/Release/lib/xamarin.android/xbuild/Xamarin/Android/lib/x86_64/libsqlite3_xamarin.so" because it was not found. 
```

**Pull request closed without merging**